torch/testing/_internal/jit_utils.py

```diff
@@ -303,6 +328,9 @@ def get_frame_vars(self, frames_up):
         defined_vars.update(frame.f_globals)
         return defined_vars
 
+    def assertRaisesRegexWithHighlight(self, exception, regex, highlight):
+        return _AssertRaisesRegexWithHighlightContext(self, exception, regex, highlight)
+
     def checkScriptRaisesRegex(self, script, inputs, exception, regex,
                                outputs=None, capture_output=False, profiling=ProfilingMode.PROFILING):
```

Comment:
What do you think about adding another argument to this to check for a highlighted token? i.e.
```
    def checkScriptRaisesRegex(self, script, inputs, exception, regex,
                               outputs=None, capture_output=False, profiling=ProfilingMode.PROFILING, highlighted_token=None):
    ...
    ...
    if highlighted_token:
        # do stuff
```

Sorry, I should have thought more about this before suggesting that you create `assertRaisesRegexWithHighlight`.

Reply:
I looked at it but checkScriptRaisesRegex doesn't do what I expected. It first runs function in normal python mode and expects an exception. But this exception generally doesn't have any highlighting. So "highlight check" seems not useful here. What do you think?

Reply:
Good point.